main/main.cpp

```diff
@@ -581,12 +581,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	GLOBAL_DEF("memory/multithread/thread_rid_pool_prealloc", 60);
 
 	GLOBAL_DEF("network/debug/max_remote_stdout_chars_per_second", 2048);
-	GLOBAL_DEF("network/debug/remote_port", 6007);
```

Comment:
Okay, what I don't understand than is why do we need it in the project setting instead of just the editor settings?

I think in the end there are 2 pairs or values:
1. Listen port, listen address (`*`,`0.0.0.0`,`::`, or a specific interface) (editor)
2. Connect port, connect address (specific interface) (project)

The connection address and port can be set passing the `-rdebug IP:PORT` command line option. On exported projects (i.e. android run) that argument is set as an export flag in the data pack from the editor address and port.
Of course, the ports and addresses must match or it won't work.

The editor does not have a button to just listen for connection, but I guess it could in the future.

Is there any case where we would want to set the connect address and port via the project settings? What would be the reason behind that?

In fact, I'm okay in separating the client and server options, I'm just afraid it might gets confusing.
Suggestions are welcome.